INN/umbrella-borderzine
# Add 6 column grid widget for photo galleries section

## Changes

This pull request makes the following changes:

- Creates the `Borderzine_6_Col_Widget` widget (copied from Largo recent posts) to display posts in rows of 6 for the photo galleries section

![Screen Shot 2019-10-07 at 4 00 31 PM](https://user-images.githubusercontent.com/18353636/66344350-9d107100-e91b-11e9-891d-a2a744e8d275.png)

## Why



For #36

## Testing/Questions

Features that this PR affects:

- Special series template


Questions that need to be answered before merging:


Steps to test this PR:

1. Add the `Borderzine 6 Column Widget` widget to the `Series Life on the Borderline: Footer` widget area
    - Title: `Photo Galleries`
    - Avoid Duplicate Posts
    - Large image
    - Excerpt display: none
    - Hide the publish date
    - Limit to category: Multimedia (?)
2. View widget on frontend and verify that it looks ok

## Additional information

INN Member/Labs Client requesting: (if applicable)
